src/app/users/getManageConsoleServices.js

```diff
@@ -0,0 +1,19 @@
+const { sendResult } = require('../../infrastructure/utils');
+const { getAllServices }= require('../../infrastructure/applications')
+const { getUserDetails } = require('./utils');
+
+
+const getManageConsoleServices = async (req, res) => {
+  const user = await getUserDetails(req);
+  const services = await getAllServices();
+
+  sendResult(req, res, 'users/views/selectManageConsoleService', {
+    // layout: 'sharedViews/layoutNew.ejs', 

```

Comment:
Was there an issue with the newLayout for why it wasn't used here? 😄 As it may be a little jarring flipping from old layout to new layout and back again 👍

Reply:
Thanks Jared, no issue with the new layout, more that the classes from the other existing table components in the app were not working with the new layout. Then the classes from the example table components from the gds website did not give a consistent look with the rest of the app.